cmd/livepeer/starter/starter.go

```diff
@@ -1412,3 +1417,31 @@ func getBroadcasterPrices(broadcasterPrices string) []BroadcasterPrice {
 
 	return pricesSet.Prices
 }
+
+type KeystorePath struct {
+	path, address interface{}
+}
+
+func ParseEthKeystorePath(ethKeystorePath string) (KeystorePath, error) {
+	var keystore KeystorePath = KeystorePath{"", ""}
+	ethKeystorePath = strings.TrimSuffix(ethKeystorePath, "/")
+
+	if fileInfo, err := os.Stat(ethKeystorePath); !os.IsNotExist(err) && fileInfo != nil {
```

Comment:
Why `!os.IsNotExist(err)` and not just `err != nil`? What should we do in case of other errors?

Reply:
There wasn't a good reason, I have now simplified this error check to `err != nil`